webui/react/src/utils/types.ts

```diff
@@ -287,3 +297,29 @@ export const getBatchNumber = (
 };
 
 export type Eventually<T> = T | Promise<T>;
+
+export const jobTypeIconName = (jobType: JobType): string => {
+  return jobType.toString().replace('TYPE_', '').toLowerCase();
+};
+
+export const jobTypeLabel = (jobType: JobType): string => {
+  return capitalize(jobTypeIconName(jobType));
+};
+
+// translate determinedjobv1type to command type
+export const determinedjobv1TypeToCommandType = (
+  determinedjobv1Type: Determinedjobv1Type,
+): CommandType | undefined => {
+  switch (determinedjobv1Type) {
+    case Determinedjobv1Type.NOTEBOOK:
+      return CommandType.JupyterLab;
+    case Determinedjobv1Type.SHELL:
+      return CommandType.Shell;
+    case Determinedjobv1Type.TENSORBOARD:
+      return CommandType.TensorBoard;
+    case Determinedjobv1Type.COMMAND:
+      return CommandType.Command;
+    default:
+      return undefined;
+  }
+};
```

Comment:
Nothing actionable. Just wondering if this would fit better under `/utils/job.ts` (and if there are other things that can go there).